regression-test/suites/datatype_p0/decimalv3/test_decimal256_cast.groovy

```diff
@@ -41,4 +41,29 @@ suite("test_decimal256_cast") {
         select cast('0.000000000000000000000000000000000000000000000000000000000000000000000012345678901' as decimalv3(76,0));
     """
 
+    sql """
+        drop table  if exists dec256cast_to_float;
```

Comment:
The previous cast test only test const literal, should also add some test to insert the data to a table and cast them during select.